src/components/grant/index.tsx

```diff
@@ -1,28 +1,35 @@
 import React from 'react'
 import { GrantTokensForm } from '../home/components/Forms/GrantTokensForm'
-import { Address, EtherBalance, TokenBalance } from '../wallet/components'
+import { Address } from '../wallet/components'
 
 const Row = ({ caption, Content }: { caption: string; Content: any }) => (
   <tr>
     <td>
-      <h2>{caption}</h2>
+      <span className="t-s t-bold">{caption}</span>
     </td>
     <td>
       <Content />
     </td>
-    <td />
   </tr>
 )
+const Heading = ({ caption, Content }: { caption: string; Content: any }) => (
+  <div>
+    <h1>{caption}</h1>
+    <h2>
+      <Content />
+      <i className="icon icon-contracts"/>
+    </h2>
+  </div>
+)
```

Comment:
This should not be copied and pasted but put in its own file and imported everywhere where it is needed.